src/Components/v2/AuctionFAQ.tsx

```diff
@@ -0,0 +1,186 @@
+import { Box, media, Serif, space, Spacer, Theme, Toggle } from "@artsy/palette"
```

Comment:
Can / should this be in [`src/App/Auction/Components`](https://github.com/artsy/reaction/tree/master/src/Apps/Auction/Components) – the `RegistrationForm` is located in that directory, so would probably be the most logical to co-locate all components for the same Auction application

Reply:
Could also be in `src/App/Auctions/Routes/FAQ` as there is an FAQ route

Reply:
Moved to the components folder.

Reply:
The pattern we follow in other apps is if the component relates to a specific route it is typically co-located in a
```
src/App/Auctions/Routes/FAQ/Components
```
folder under the route. If it's more general or free floating (like maybe the FAQModal indeed is) then `src/App/Auctions/Components`.